src/software/time/time.h

```diff
@@ -23,7 +23,7 @@ class Time
     /**
      * The default constructor for a Time. Creates a Time at time 0
      */
-    Time() = delete;
+    Time();
```

Comment:
In the original ticket (#1683) it was mentioned that the default Time constructor should be deleted, however deleting it caused some failed tests since this constructor is being used in `time.cpp` [file](https://github.com/UBC-Thunderbots/Software/blob/bedeb7efccd7bcf6e641c0ae2a77802bf2e02c5d/src/software/time/time.cpp#L5): `Time::Time() : time_in_seconds(0) {}`. So I reverted my changes back to how it was originally.